src/types/validators.go

```diff
@@ -74,6 +75,36 @@ func (p *Project) assignDefaultProcessValues() {
 	}
 }
 
+func (p *Project) validateProcessCommand() {
+	for name, proc := range p.Processes {
+		command := proc.Command
+		entrypoint := proc.Entrypoint
+
+		if command != "" {
+			if len(entrypoint) > 0 {
+				message := fmt.Sprintf("Both command and entrypoint are set! Using command and ignoring entrypoint (procces: %s)", name)
+				fmt.Println(message)
+				log.Error().Msg(message)
+			}
+
+			proc.Executable = p.ShellConfig.ShellCommand;
+			proc.Args = []string{
+				p.ShellConfig.ShellArgument,
+				command,
+			}
+		} else if len(entrypoint) > 0 {
+			proc.Executable = entrypoint[0]
+			proc.Args = entrypoint[1:]
+		} else {
+			message := fmt.Sprintf("Either command or entrypoint need to be non-empty (procces: %s)", name)
+			fmt.Println(message)
+			log.Fatal().Msg(message)
```

Comment:
just realised that this is a breaking change, since previously empty command was allowed

Reply:
You are right, ignore the empty command for now. I will fix it in the Strict mode.